src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13684,8 +13642,6 @@ void VRFCondenserEquipment::VRFOU_CompSpd(
     static constexpr std::string_view RoutineName("VRFOU_CompSpd");
 
     // variable initializations: component index
-    TUListNum = this->ZoneTUListPtr;
-    NumTUInList = state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).NumTUInList;
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13688]:(style),[unreadVariable],Variable 'NumTUInList' is assigned a value that is never used.